CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 
 ## Getting Started
 
-Refer to the [readme](README.md) for installation and development instructions.
+Refer to the Dash User Guide for [installation](https://dash.plot.ly/installation) instructions.
```

Comment:
The installation instructions on the dash docs is for developing dash apps. 
Installation for development on the dash repo should be:

- Create a virtualenv: 
`python -m venv venv` or `virtualenv venv` for python 2.
- Activate the virtualenv:
`. venv/bin/activate` or `venv\scripts\activate` for windows.
- Install the dev dependencies:
`pip install -r .circleci/requirements/dev-requirements.txt`

Reply:
Thanks for reviewing, @T4rk1n! In light of your comment, I have submitted another change, which is more to the point. I don't think it is healthy to promote the use of Python 2 any more (I'll share references if you ask for them), so I have only included the Python 3 way.